fossunited/www/grants/grantees/rss.py

```
@@ -0,0 +1,137 @@
+                "link": url or host + "/grants/grantees",
+                "author": "",
+                "published_date": format_datetime(frappe.utils.get_datetime(date)),
+                "modified": g.modified,
+                "category": escape_html(g.grant_type),
+                "content": (
+                    f"<![CDATA["
+                    f"<p><strong>Grant Type:</strong> {escape_html(g.grant_type)}</p>"
+                    f"<p><strong>Amount:</strong> {escape_html(amount)}</p>"
+                    f"{description}"
+                    + (f'<p><a href="{url}">Project Website →</a></p>' if url else "")
+                    + f"]]>"
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)